### ISSUE TITLE
AttributeError in copy_clipboard() when I rightclicking the link and selecting "Copy"

### ISSUE THREAD

**Issue by Rattenkrieg** [Guake/guake#1311]
```
File "/usr/lib/python3/dist-packages/guake/guake_app.py", line 1351, in accel_copy_clipboard
    self.notebook.get_current_terminal().copy_clipboard()
File "/usr/lib/python3/dist-packages/guake/terminal.py", line 140, in copy_clipboard
    guake_clipboard = Gtk.Clipboard.get_default(self.window.get_display())
AttributeError: 'GuakeTerminal' object has no attribute 'window'
```
Guake version: 3.0.5-1
Distro: Ubuntu 18.04
Env: Python 3.6.5-3; GNOME Shell 3.28.1; Xorg 1:7.7+19ubuntu7

**Comment by gsemet** [comment]
Can you reproduce with latest version (compile from source?) should be fixed on HEAD

**Comment by Rattenkrieg** [comment]
The issue is reproducible after compiling from source. However I've lost access to that Ubuntu machine and did compilation on Fedora 28, GNOME Shell 3.28.2.

**Comment by gsemet** [comment]
can you check if b324279 fixed this issue?

**Comment by abcarroll** [comment]
I attempted to reproduce the original issue as reported by @Rattenkrieg for the sake of answering that question but was unable to.

However, in completely unrelated testing, for some reason, b324279 has a bizarre side-effect for me of making drop-downs and other parts of the preferences window show up strange and broken.

Screenshots:

With b324279:
![guake-with-b324279](https://user-images.githubusercontent.com/961513/40155931-5d93c646-5964-11e8-9586-9a62aaa7bee2.png)

Without b324279 (HEAD currently at 738d503 with it git revert'd)
![guake-without-b324279](https://user-images.githubusercontent.com/961513/40155962-8e17f0c6-5964-11e8-9c33-0c4454d67552.png)

**Comment by Rattenkrieg** [comment]
I'm at current master - 2908357 and have following error when performing "right click -> copy" on the link
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/guake/guake_app.py", line 1401, in accel_copy_clipboard
    self.notebook.get_current_terminal().copy_clipboard()
  File "/usr/local/lib/python3.6/site-packages/guake/terminal.py", line 138, in copy_clipboard
    guake_clipboard.set_text(self.matched_value)
TypeError: Gtk.Clipboard.set_text() takes exactly 3 arguments (2 given)
```

GDM 3.28.2
Gtk: 3.22.30

**Comment by gsemet** [comment]
seems pretty legit, I'll investigate it

**Comment by robert-matusewicz** [comment]
I'm on 3.6.3 and I'm not able to reproduce this issue. Looking at the stacktrace, it was fixed in 6bc9e53

**Comment by Davidy22** [comment]
Also unable to reproduce, closing.
